src/reconcile/rec_row.c

```diff
@@ -785,6 +785,13 @@ __wt_rec_row_leaf(
         /* Build value cell. */
         dictionary = false;
         if (upd == NULL) {
+            /* Check the visibility of the on disk key before writing. */            
```

Comment:
If we reconcile an update with a globally visible stop time pair AND there are no new updates for that key? Is this for the case where you don't touch a key but its previously written stop pair becomes globally visible while it's in cache?

Either way, could you please elaborate a bit more on the comment?

Reply:
Comment has updated to provide a better explanation.